Alpaca.Markets/AlpacaTradingClient.General.cs

```diff
@@ -154,6 +154,39 @@ public sealed partial class AlpacaTradingClient
                 _alpacaRestApiThrottler, builder, cancellationToken);
         }
 
+        /// <summary>
+        /// Gets portfolio equity history from Alpaca REST API endpoint.
+        /// </summary>
+        /// <param name="startDate">Start date for desired history.</param>
+        /// <param name="endDate">End date for desired history. Default value is today. </param>
+        /// <param name="period">Period value for desired history. Default value is 1 month.</param>
+        /// <param name="timeframe">Timeframe value for desired history. Default value is 1 minute for a period shorter than 7 days, 15 minutes for a period less than 30 days, or 1 day for a longer period.</param>
+        /// <param name="extendedHours">If true, include extended hours in the result. This is effective only for timeframe less than 1 day.</param>
+        /// <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
+        /// <returns>Read-only portfolio history information object.</returns>
+        public Task<IPortfolioHistory> GetPortfolioHistoryAsync(
+                DateTime? startDate,
+                DateTime? endDate,
+                HistoryPeriod? period,
+                TimeFrame? timeframe,
+                Boolean? extendedHours,
+                CancellationToken cancellationToken = default)
+        {
+            var builder = new UriBuilder(_httpClient.BaseAddress)
+            {
+                Path = _httpClient.BaseAddress.AbsolutePath + "account/portfolio/history",
+                Query = new QueryBuilder()
+                    .AddParameter("start_date", startDate, "O")
```

Comment:
According to documentation (and names), both parameters should be formatted using `YYYY-MM-DD` format string not using the full ISO date/time format.